docs/stories/Inputs/Checkbox.stories.tsx

```diff
@@ -2,12 +2,19 @@ import * as React from 'react';
 
 import { useArgs } from '@storybook/preview-api';
 import { Meta, StoryObj } from '@storybook/react';
-import { Checkbox, Field, FieldHint, FieldError } from '@strapi/design-system';
+import { Checkbox, Field, FieldHint, FieldError, Flex } from '@strapi/design-system';
 import { default as outdent } from 'outdent';
 
 const meta: Meta<typeof Checkbox> = {
   title: 'Inputs/Checkbox',
   component: Checkbox,
+  decorators: [
```

Comment:
I think this is applicable to other components as well, can we have it for stories in preview?

Reply:
sure!

Reply:
Actually upon reflection, i think this needs to be tested better because it shows that some inputs don't grow to full width in certain conditions. We need to make some `Patterns` e.g. a form, to investigate the right CSS to make this work without breaking other things 🤔  i'm gonna park it but write a follow up.

Reply:
ok 👍